src/wrappers/tauri-api-stubs/event.ts

```diff
@@ -0,0 +1,24 @@
+/** Stubs for @tauri-apps/api/event */
+
+import * as tauriEvent from '@tauri-apps/api/event';
+
+const listeners = new Map<string, tauriEvent.EventCallback<unknown>>();
```

Comment:
@danvk added support to register multiple listeners for the same event. I'm not sure if we are using that currently, but def something we might use in the future.